packages/editor/src/components/assets/useUpload.tsx

```diff
@@ -32,9 +32,6 @@ import { getState } from '@etherealengine/hyperflux'
 
 import { getEntries, uploadProjectAssetsFromUpload } from '../../functions/assetFunctions'
 import { EditorState } from '../../services/EditorServices'
-import { DialogState } from '../dialogs/DialogState'
-import ErrorDialog from '../dialogs/ErrorDialog'
-import { ProgressDialog } from '../dialogs/ProgressDialog'
```

Comment:
All of these were not functioning in the new UI. Also, a full refactor of the useUpload function is needed.